ruleset.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<ruleset>
+    <!-- Contributte Coding Standard -->
+    <rule ref="./vendor/ninjify/coding-standard/ruleset.xml"/>
+
+    <!-- Specific rules -->
+    <!-- <rule ref="SlevomatCodingStandard.Files.TypeNameMatchesFileName">
+        <properties>
+            <property name="rootNamespaces" type="array" value="
+                app/modules/FrontModule/component=>App\FrontModule\Components,
+                app/modules/FrontModule=>App\FrontModule,
+                app/libs/Model=>App\Model,
+                app/libs/Core=>App\Core,
+                app/libs/Aprila=>Aprila,
+				app=>App
```

Comment:
Je tu vsude mix tabs vs space. Defakto jedina vec kvuli ktere se to dela :) jde to nejak ohlidat? Fixnout jak v php tak tu? tak v latte apd?

Reply:
klidne i pridat ten editor config, aby ho to naucilo psat ty veci rovnou spravne?

Reply:
Koukám že je tam mix :(

Latte jsem CodeSnifferem nikdy nekontroloval, popravdě ani nevím jestli to jde.

Reply:
tak jen to php by bylo super at je konzistentni ... zbytek udelam rucne a pridal bych ten editor config

Reply:
Jop, moje chyba, omylem jsem smazal zákaz tabů když jsem tu čachroval s commity, původně to tam bylo

Reply:
chapu, sam nechapu jak s tim pracujes, ja bych to vse delal pokazde znova, tak doufam ze jsi vetsi git master a mas v tom trosku nejakej system ... ps: rad bych si to pak nekdy nechal ukazat jak se takove veci pekne delaji.

Reply:
Vyzkoušeno, codesniffer latte seežere, teda sežere, ale nic nenajde.

Reply:
👌